Tests/SwiftFormatPrettyPrintTests/MacroCallTests.swift

```diff
@@ -0,0 +1,117 @@
+import SwiftFormatConfiguration
+
+final class MacroCallTests: PrettyPrintTestCase {
+  func testNoWhiteSpaceAfterMacroWithoutTrailingClosure() {
+    let input =
+      """
+      func myFunction() {
+        print("Currently running \\(#function)")
+      }
+      
+      """
+
+    let expected =
+      """
+      func myFunction() {
+        print("Currently running \\(#function)")
+      }
+
+      """
+
+    assertPrettyPrintEqual(input: input, expected: expected, linelength: 50)
+  }
+
+  func testKeepWhiteSpaceBeforeTrailingClosure() {
+    let input =
+      """
+      #Preview {}
+      #Preview("MyPreview) {
```

Comment:
Here and throughout below, it looks like you're missing the trailing quote after `"MyPreview`, so we can't really trust the test results (if they do pass, the parser may be fixing the literal for us automatically)

Reply:
Thanks for the thorough attention, I would have totally missed that! Sorry for the inconvenience 🙇
The strings should now properly be closed with the latest update.